src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_instantiate.yaml

```
@@ -0,0 +1,51 @@
+post:
+  summary: Instantiate a new package through a project link.
+  description: |
+    Instantiate a new package in the specified project based on the packages inherited
+    through a linked project (local). The new package will be a branch of the package
+    that lives in the linked project unless the `makeoriginolder` parameter is provided.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: makeoriginolder
+      schema:
+        type: string
+      description: |
+        Instead of creating a branch of the package, it will instantiate a copy
+        of the package and extend the `vrev` to ensure the copy represents a
+        newer version of the package.
+  responses:
+    '200':
+      description: ok
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          example:
+            code: ok
+            summary: Ok
```

```suggestion
      $ref: '../components/responses/succeeded.yaml'
```
We could use the already defined "succeeded" response here, if I'm not wrong.

---

True, forgot we have it already. Done